web/hooks/use-group.ts

```diff
@@ -205,3 +207,18 @@ export function useGroupContractIds(groupId: string) {
 export function useGroups(groupIds: string[]) {
   return useStoreItems(groupIds, listenForGroup, { loadOnce: true })
 }
+
+export function useIsGroupMember(groupSlug: string) {
+  const [isMember, setIsMember] = useState<any>(undefined)
+  const user = useUser()
+  useEffect(() => {
+    if (user) {
+      setTimeout(() => {
```

Comment:
Other people that use this hook might not want a delay haha, probably should either rename the hook, or add a param to put the delay.

Also, I'm skeptical of delays like this. People want access ASAP. An unlocking animation might be the kind of thing you'd want to see once.

Reply:
Yeah I can move it out, it's mainly for refreshes on the group page, or else I get an error that's like "must be logged in to call API"